src/partials/_mediaQueries.scss

```diff
@@ -0,0 +1,44 @@
+@use "./variables" as *;
+
+@media (max-width: 1100px) {
+  html {
+    font-size: 100%;
+  }
+  .thoughtForm {
+    max-width: 100%;
+    .Container {
+      padding: 2rem;
+      label {
+        font-size: 1.25rem;
+      }
+      p {
+        font-size: 1rem;
+        width: 100%;
+      }
+    }
+  }
+  .storedThoughts ul li {
```

Comment:
the user name is not showing up properly (overflow and breaks into 2 lines ) below 1400px

Reply:
Fixed.